src/Templates/NServiceBusWindowsService/Host.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+namespace NServiceBusWindowsService
+{
+    internal class Host
+    {
+        // TODO: optionally choose a custom logging library
+        // https://docs.particular.net/nservicebus/logging/#custom-logging
+        // LogManager.Use<TheLoggingFactory>();
+        private static readonly ILog log = LogManager.GetLogger<Host>();
+
+        private IEndpointInstance endpoint;
+
+        // TODO: give the endpoint an appropriate name
+        public string EndpointName => "MyNServiceBusWindowsService";
```

Comment:
Shouldn't this be a get only property instead of a public field? e.g.:

```
public string EndpointName{ get; } => "MyNServiceBusWindowsService";
```

Reply:
`public string Foo => "Bar";` is already a get-only property. That is why there is an expression rather than an assignment. `public string Foo { get; } => "Bar";` is invalid syntax. You can have `public string Foo { get; } = "Bar";` but that is an initial assignment, rather than an expression body.

Reply:
yeah, I was writing from the phone and I just copy / pasted adding the `{get;}`.

> `public string Foo => "Bar";` is already a get-only property.

Now I have one more reason to dislike expression bodied members 😄